tests/release/e2e-test-default-with-deployment.go

```diff
@@ -162,40 +181,40 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1199]test-release-e2e-with-deploy
 					return false
 				}
 
-				return releaseCreated.HasStarted() && releaseCreated.IsDone() && releaseCreated.Status.Conditions[0].Status == "True"
+				return releaseCreated.IsReleased()
 			}, releaseCreationTimeout, defaultInterval).Should(BeTrue())
 		})
 
-		It("tests that copying application and component works as designed.", func(ctx SpecContext) {
-			workingDir, err := os.Getwd()
-			if err != nil {
-				GinkgoWriter.Printf(err.Error())
-			}
-
-			// Download copy-applications.sh script from release-utils repo
-			args := []string{"https://raw.githubusercontent.com/hacbs-release/release-utils/main/copy-application.sh", "-o", "copy-application.sh"}
-			err = utils.ExecuteCommandInASpecificDirectory("curl", args, workingDir)
-			Expect(err).NotTo(HaveOccurred())
-
-			args = []string{"775", "copy-application.sh"}
-			err = utils.ExecuteCommandInASpecificDirectory("chmod", args, workingDir)
-			Expect(err).NotTo(HaveOccurred())
-
-			// Copying application in dev namespace to managed namespace
-			args = []string{managedNamespace, "-a", devNamespace + "/" + applicationNameDefault}
-			err = utils.ExecuteCommandInASpecificDirectory("./copy-application.sh", args, workingDir)
-			Expect(err).NotTo(HaveOccurred())
-		}, SpecTimeout(snapshotCreationTimeout+namespaceCreationTimeout*2))
+		//It("tests that copying application and component works as designed.", func(ctx SpecContext) {
```

Comment:
Are these comments supposed?

Reply:
Thanks @rsoaresd. I forgot to remove it (the actual code was moved to the top of the file).